pkg/core/blockchain.go

```diff
@@ -1254,20 +1253,20 @@ func (bc *Blockchain) verifyTxAttributes(tx *transaction.Transaction) error {
 		case transaction.HighPriority:
 			pubs, err := bc.contracts.NEO.GetCommitteeMembers(bc, bc.dao)
 			if err != nil {
-				return fmt.Errorf("%w: %v", ErrInvalidAttribute, err)
+				return fmt.Errorf("%w: %v", transaction.ErrInvalidAttribute, err)
 			}
 			m := len(pubs) - (len(pubs)-1)/2
 			s, err := smartcontract.CreateMultiSigRedeemScript(m, pubs)
 			if err != nil {
-				return fmt.Errorf("%w: %v", ErrInvalidAttribute, err)
+				return fmt.Errorf("%w: %v", transaction.ErrInvalidAttribute, err)
 			}
 			h := hash.Hash160(s)
 			for i := range tx.Signers {
 				if tx.Signers[i].Account.Equals(h) {
 					return nil
 				}
 			}
-			return fmt.Errorf("%w: signers does not contain committee", ErrInvalidAttribute)
+			return fmt.Errorf("%w: signers does not contain committee", transaction.ErrInvalidAttribute)
```

Comment:
Not sure we should return `transaction.*` errors from `core`. If we're to do that we might as well not mess with the ones returned by `mempool`, but then the receiver would have to import those to distinguish between different errors.

Reply:
Fixed.